src/utils/util.ts

```diff
@@ -1,3 +1,21 @@
+import { Position } from '../@types';
+
+export type Shape = {
+    height: number;
+    width: number;
+    x: number;
+    y: number;
+};
+
+export const centerShape = (shape: Shape): Position => {
```

Comment:
I think we should really aim to have as few global utilities as possible, to the point where we should try to eliminate this file and folder entirely.

Reply:
As discussed, will duplicate this function and look into getting rid of this file entirely in a follow up PR